src/main.rs

```diff
@@ -128,26 +130,26 @@ fn main() -> Result<()> {
             .to_lowercase()
     });
 
+    // Generate the temporary project
     let mut command = process::Command::new("cargo");
-
     command
         .current_dir(&tmp_dir)
         .args(&["init", "--name", project_name.as_str()]);
-
     if cli.lib {
         command.arg("--lib");
     }
-
     if !command.status().context("Could not start cargo")?.success() {
         bail!("Cargo command failed");
     };
 
+    // Generate the config file
     let delete_file = tmp_dir.path().join("TO_DELETE");
     fs::write(
         &delete_file,
         "Delete this file if you want to preserve this project",
     )?;
```

Comment:
This is really not what this code does :thinking:

Reply:
was too late, fixed